src/platform_impl/macos/menu.rs

```diff
@@ -113,3 +238,64 @@ fn menu_item(
         item
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct Menu {
+    pub(crate) raw: IdRef,
+}
+
+impl Menu {
+    pub fn new() -> Self {
+        unsafe {
+            Self {
+                raw: IdRef::new(NSMenu::new(nil)),
+            }
+        }
+    }
+
+    pub fn add_item<S: Into<String>, H: Into<Option<Hotkey>>>(
+        &mut self,
+        id: usize,
```

Comment:
We ask for `usize` here for the `id`, but the `MenuEntryActivated` event has an `isize`. Is there any reason for that inconsistency?

Reply:
I think the most consistent way would be to use `u32`, since `cocoa` uses `isize` and `winapi` can use `usize` and `u32`.
This is probably a leftover of my experimentation with mapping the `id` to a `Message`.